samples/tutorials/Data passing in python components/Data passing in python components - Files.py

```diff
@@ -0,0 +1,149 @@
+# %% [markdown]
```

Comment:
I'm curious how can one open the .py file and apply this md magic can be properly rendered?

Reply:
VS Code supports converting both ways between Jupyter and Python: https://code.visualstudio.com/docs/python/jupyter-support-py#_export-a-jupyter-notebook

P.S. If you think that these comment magics should not be in the file, I can remove them.

Reply:
No worry, you can leave as it is